### PR TITLE
fix:ConsumerGroup bug while removeTopics

### PR DESCRIPTION
**Pull request by jnotnull**
fix bug: https://github.com/SOHU-Co/kafka-node/issues/999

### CODE REVIEW

**Comment by hyperlink** [comment]
`removeTopics` should work similarly to `addTopics` and let the coordinator know. See how PR #914 was implemented.

**Comment by jnotnull** [comment]
Thanks for your comments.

File: lib/consumerGroup.js [hunk 1]
```diff
@@ -660,6 +660,28 @@ ConsumerGroup.prototype.addTopics = function (topics, cb) {
...
+  topics = typeof topics === 'string' ? [topics] : topics;
+
+  async.series([
+    callback => this.client.topicExists(topics, callback),
+    callback => (this.options.autoCommit && this.generationId != null && this.memberId)
+        ? this.commit(true, callback)
+        : callback(null),
+    callback => this.leaveGroup(callback),
+    callback => {
+      this.topics = this.topics.filter(function (topic) {
```
Comment (hyperlink) [inline lib/consumerGroup.js]: use `_.difference` instead ?

Comment (jnotnull) [reply lib/consumerGroup.js]: done

**Comment by hyperlink** [comment]
Published as 3.0.0 thanks again.

**Pull request merged**